launcher/src/main/java/com/skcraft/launcher/swing/WebpagePanel.java

```diff
@@ -249,6 +249,7 @@ public void run() {
             try {
                 conn = (HttpURLConnection) url.openConnection();
                 conn.setRequestMethod("GET");
+                conn.addRequestProperty("User-Agent", "");
```

Comment:
This should be pulled from https://github.com/SKCraft/Launcher/blob/master/launcher/src/main/resources/com/skcraft/launcher/launcher.properties

Example (and not a dev just jotting it down);
`userAgent=SKCraft`
And here something like `conn.addRequestProperty("User-Agent", $userAgent);`
This would make PR's cleaner later on down the track

That way if someone wishes to use the header for stats they can

Reply:
Yeah I support this idea. Makes it customizable.

Reply:
I was thinking about the customizability of the property as well. Thanks for the feedback.

For reference, User-Agent should almost always be specified in the request header: [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.5.3).

I also noticed that the property is not set in the `execute` method of the `HttpRequest` objects (in the launcher & launcher-bootstrap modules). The User-Agent will then have to be specified in both the [`launcher.properties`](https://github.com/SKCraft/Launcher/blob/master/launcher/src/main/resources/com/skcraft/launcher/launcher.properties) and [`bootstrap.properties`](https://github.com/SKCraft/Launcher/blob/master/launcher-bootstrap/src/main/resources/com/skcraft/launcher/bootstrap.properties). Though, as @spannerman79 said, someone can then distinguish been the two.